plugins/de.cognicrypt.codegenerator/src/main/java/de/cognicrypt/codegenerator/crysl/templates/encryptionhybrid/SecureEncryptor.java

```diff
@@ -1,5 +1,5 @@
 /********************************************************************************
- * Copyright (c) 2015-2019 TU Darmstadt, Paderborn University
+ * Copyright (c) 2015-2021 TU Darmstadt, Paderborn University

```

Comment:
I was trying to generate this encryption hybrid template through CogniCrypt GEN, but did not find the right combination of options to do so. Which options shall I choose to do so and test its generation? Thanks

Reply:
Thanks, @enriozuni, for reviewing my PR. In order to reach this, you have to answer the first question "Unencrypted digital channel (e.g. email) and the second one Byte array or other.